SimCalorimetry/EcalEBTrigPrimAlgos/src/EcalEBTrigPrimTestAlgo.cc

```diff
@@ -32,26 +32,31 @@
 
 
 
-#include <TTree.h>
-#include <TMath.h>
-
-
 //----------------------------------------------------------------------
 
-const unsigned int EcalEBTrigPrimTestAlgo::nrSamples_=5;
-const unsigned int EcalEBTrigPrimTestAlgo::maxNrTowers_=2448;
-const unsigned int EcalEBTrigPrimTestAlgo::maxNrSamplesOut_=10;
+//const unsigned int EcalEBTrigPrimTestAlgo::nrSamples_=5;
+//const unsigned int EcalEBTrigPrimTestAlgo::maxNrTowers_=2448;
+//const unsigned int EcalEBTrigPrimTestAlgo::maxNrSamplesOut_=10;
 
 
-EcalEBTrigPrimTestAlgo::EcalEBTrigPrimTestAlgo(const edm::EventSetup & setup,int nSam, int binofmax,bool tcpFormat, bool barrelOnly,bool debug, bool famos): 
-  nSamples_(nSam),binOfMaximum_(binofmax), tcpFormat_(tcpFormat), barrelOnly_(barrelOnly), debug_(debug), famos_(famos)
+EcalEBTrigPrimTestAlgo::EcalEBTrigPrimTestAlgo(const edm::EventSetup & setup,int nSam, int binofmax,bool tcpFormat, bool barrelOnly,bool debug, bool famos)
 
 {
 
- maxNrSamples_=10;
- this->init(setup);
+  nSamples_=nSam;
+  binOfMaximum_=binofmax;
+  tcpFormat_=tcpFormat;
+  barrelOnly_=barrelOnly;
+  debug_=debug; 
+  famos_=famos;
+  maxNrSamples_=10;
+  this->init(setup   );
+  fenixTcpFormat_ = new EcalFenixTcpFormat(tcpFormat_, debug_, famos_, binOfMaximum_);
+  tcpformat_out_.resize(maxNrSamples_);   
+
 }
 
+/*
```

Comment:
delete commented-out code

Reply:
Please do, or justify why you prefer not doing it